src/domain/validation-constants.js

```diff
@@ -7,6 +7,8 @@ export const maxDocumentCommentTextLength = 2000;
 
 export const maxDocumentRevisionCreatedBecauseLength = 200;
 
+export const maxDocumentCategoryNameLength = 25;
```

Comment:
This we should also use in the `documentCategoryDbSchema` validation (which is not included in your branch yet). Just as a reminder.

Reply:
Good observation, I rebased from `main` and included it there too 👍

Reply:
That is even better than just a reminder :+1: